Sources/StreamVideo/WebRTC/WebRTCClient.swift

```diff
@@ -1311,9 +1311,10 @@ class WebRTCClient: NSObject, @unchecked Sendable {
     
     private func addOnParticipantsChangeHandler() {
         Task { [weak self] in
+            guard let self else { return }
```

Comment:
I think I already fixed this in another PR.

Reply:
I didn't find somewhere where we are actually terminating it so adding the fix here.